shell/platform/windows/win32_window.cc

```diff
@@ -198,6 +265,35 @@ Win32Window::HandleMessage(UINT const message,
                 static_cast<double>(WHEEL_DELTA)),
                0.0);
       break;
+    case WM_INPUTLANGCHANGE:
+      // TODO(cbracken): pass this to TextInputManager to aid with language-specific issues.
+      break;
+    case WM_IME_SETCONTEXT:
+      OnImeSetContext(message, wparam, lparam);
+      result_lparam &= ~ISC_SHOWUICOMPOSITIONWINDOW;
+      break;
+    case WM_IME_STARTCOMPOSITION:
+      OnImeStartComposition(message, wparam, lparam);
+      // Suppress further processing by DefWindowProc() so that the default
+      // system IME style isn't used, but rather the one we set in the
+      // WM_IME_SETCONTEXT handler.
+      return TRUE;
+    case WM_IME_COMPOSITION:
+      OnImeComposition(message, wparam, lparam);
+      if (lparam & GCS_RESULTSTR) {
+        // Suppress further processing by DefWindowProc() since otherwise it
+        // will emit the result string as WM_CHAR messages on commit. Instead,
+        // we handle committing the composing text to the text ourselves in the
```

Comment:
Please reword these comments to avoid "we", and instead be specific about what is doing this. I'm not sure what specific piece of code is being personified here (which is a common problem with "we" in comments).

Reply:
Done.